signal_scanner_bot/twitter.py

```diff
@@ -0,0 +1,39 @@
+import logging
+from datetime import datetime
+from textwrap import dedent
+
+import tweepy
+
+from . import env
+
+log = logging.getLogger(__name__)
+
+HASHTAGS = ["#SeattleProtestComms", "#SeaScanner", "#SeattleProtests"]
```

Comment:
Has the use of #SeattleProtests hashtag been cleared for this purpose? I know that there were some originally some concerns about scanners using this hashtag about a month ago.

Reply:
Hmm, fair question! I've seen some prominent scanners using this hashtag recently, but I'll keep an eye out for what people are saying and change these as needed.